cvFit/src/app/form/contactUs.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from "@angular/core";
+
+
+@Component({
+    selector: 'app-contactUs',
+    templateUrl: './contactUs.component.html'
+})
+export class ContactUsComponent {
+    firstName: any;
```

Comment:
you always want to type your variables. Typically if you see `any` it should probably be changed to be the actual type it is meant to be.

Reply:
Go through and add types to variables in any files you've made that are currently added as `any`.

Reply:
Will go through and change those now

Reply:
I was able to work around the issue by changing what the function received.  Still unsure why accessing variables with formValues.firstName was giving the error that firstName wasn't of type string.  Potentially something I don't understand about html form inputs.